ui/src/components/Views/Configs/ConfigPage.tsx

```diff
@@ -19,7 +19,7 @@ const useStyles = makeStyles((theme: Theme) => createStyles({
 export const ConfigPage = (): ReactElement => {
 
   const { runConfig, deleteConfig } = useRunConfig();
-  const { deleteMountPointData } = useMountPoint();
+  const data = useMountPoint();
```

Comment:
seems like this should rather be `mountPoint` or something else, calling `setMountPointData` on a `data` object feels weird